libraries/standard/http/src/http_client.c

```diff
@@ -113,28 +129,27 @@ static HTTPStatus_t getFinalResponseStatus( HTTPParsingState_t parsingState,
  *
  * @param[in] pTransport Transport interface.
  * @param[in] pResponse Response message to receive data from the network.
+ * @param[in] pRequestHeaders Request headers to extract the request method from.
```

Comment:
Request headers to extract the request method from. --> Request headers for the corresponding HTTP request.